src/app/helptext/task-calendar/replication.ts

```diff
@@ -0,0 +1,155 @@
+import { T } from '../../translate-marker';
+
+export default {
+    name_placeholder: T('Name'),
+    name_tooltip: T('Replication task name'),
+
+    direction_placeholder: T('Direction'),
+    direction_tooltip: T('Direct the flow of the ZFS snapshots. Choose between pushing\
+ snapshots to a remote host or pulling snapshots from a remote host.'),
+
+    transport_placeholder: T('Transport'),
+    transport_tooltip: T('Method for snapshot transfer:<ul>\
+ <li><i>SSH</i> is supported by most systems. It requires a previously created\
+ <a href="%%docurl%%/system.html%%webversion%%#ssh-connection" target="_blank">SSH connection</a>.</li>\
+ <li><i>SSH+NETCAT</i> uses SSH to establish a connection to the remote system, then uses\
+ an unencrypted data stream to improve data transfer speeds. This is only an option when\
+ replicating to a FreeBSD system that has <a href="https://github.com/freenas/py-libzfs"\
+ target="_blank">py-libzfs</a> installed.</li>\
+ <li><i>LOCAL</i> replicates snapshots to another dataset within the system.</li>\
+ <li><i>LEGACY</i> uses the replication engine implemented in FreeNAS 11.2 and earlier.</li></ul>'),
+
+    ssh_credentials_placeholder: T('SSH Connection'),
+    ssh_credentials_tooltip: T('Configured SSH connection to use for the replication.\
+ Choose from a list of connections configured in System > <a href="%%docurl%%/system.html%%webversion%%#ssh-connection"\
```

Comment:
We should not refer directly to menu selections, but sections of the manual are okay.  However, we do not have an ssh-connection section yet.

Reply:
the PR #2118 is to add the ssh-connection section.